deepchem/models/tensorgraph/layers.py

```diff
@@ -77,30 +97,64 @@ def _create_tensor(self):
 
 class Dense(Layer):
 
-  def __init__(self, out_channels, activation_fn=None, **kwargs):
+  def __init__(
+      self,
+      out_channels,
+      activation_fn=None,
+      biases_initializer=tf.zeros_initializer,
+      weights_initializer=tf.contrib.layers.variance_scaling_initializer,
+      time_series=False,
+      scope_name=None,
+      reuse=False,
+      **kwargs):
+    super(Dense, self).__init__(**kwargs)
     self.out_channels = out_channels
     self.out_tensor = None
     self.activation_fn = activation_fn
-    super(Dense, self).__init__(**kwargs)
+    self.biases_initializer = biases_initializer
+    self.weights_initializer = weights_initializer
+    self.time_series = time_series
```

Comment:
You apply the dense network one at a time to each element of a "Batch".  This is generally done for time series data like videos.